admin/basecalendar-admin.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * BaseCalendar admin
+ *
+ * @package WordPress
+ * @subpackage Combined Calendar
+ * @since 1.0.0
+ * @version 1.0.0
+ */
+
+
+/**
+ * general_setting_page
+ */
+function general_setting_page() { ?>
```

Comment:
I can't guess what this function will do.
That's why I guess it's better to change function name to be like `output_setting_page_html`.